WordPress/Classes/ViewRelated/Suggestions/SuggestionsTableView.swift

```diff
@@ -133,7 +133,7 @@ extension SuggestionType {
 
     // MARK: - Private
 
-    private func searchResults(searchQuery: String, suggestions: [Any], suggestionType: SuggestionType) -> [Any] {
+    func searchResults(searchQuery: String, suggestions: [Any], suggestionType: SuggestionType) -> [Any] {
```

Comment:
Could we make this static or should it stay with the instance?

Reply:
I think it's better if we make it static. But for this PR, I didn't want to make changes to `SuggestionsTableView` to prevent bugs. Especially since we are pressed for time.

I understand that changing that method to `static` is a minor change but you never know what could go wrong!

As you can see, the only change I added is making those methods `internal` because it was absolutely necessary to test the methods.

How about we make them `static` in another PR? What do you think?

Reply:
Sounds good!